cves/CVE-2016-8740.yml

```diff
@@ -98,8 +110,12 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
   answer:
+    The last test was written 5 years prior to this error, and most other tests were written over 10 years
+    prior. this was not properly tested nor was it updated following the error,
   code:
+    false
   fix:
+    false
```

Comment:
Capitalize this. False should be same line not new line

Reply:
just for consistency, I like to keep all text on new line, in addition, as false it technically not a string, but really a boolean, i think lowercase false is fine.